Client/JavaScripts/LongPress.js

```diff
@@ -0,0 +1,57 @@
+function findTagsElementAtPoint(x,y) {
```

Comment:
All new files need to include the Mozilla license header, so please add that here (copy it from any other Swift file).